### ISSUE TITLE
Fix navbar misalignment

### ISSUE THREAD

**Issue by jackiehluo** [coreatcu/website#3]
The navbar's still just maybe a pixel or two off when it should be fixed to the top of the page! Not sure why, but right now you can see the background content above it when you scroll.

![](https://cloud.githubusercontent.com/assets/8452682/9286963/82d69432-42cf-11e5-8154-2654173ae697.png)

**Comment by freeslugs** [comment]
@jackiehluo I'm not seeing this on my computer. Can you tell me what domain you're hitting, as well as what browser and OS?

**Comment by jackiehluo** [comment]
@freeslugs Yeah, so weird! It happens on both my Macbook and iMac, both currently on El Capitan. It's in Chrome and only happens on Initiative/Team.
